upgrade_test.py

```diff
@@ -420,6 +423,43 @@ def fill_and_verify_db_data(self, note, pre_fill=False, rewrite_data=True):
             self.log.info('Re-Populate DB with many types of tables and data')
             self.fill_db_data()
 
+    def fill_and_verify_alternator_db_data(self, node, note, pre_fill=False, rewrite_data=True):
+        items, items2 = [], []
+        schema = alternator.enums.YCSVSchemaTypes.HASH_AND_RANGE.value
+        schema_keys = [key_details["AttributeName"]
+                       for key_details in alternator.schemas.ALTERNATOR_SCHEMAS[schema]["KeySchema"]]
+        key_format = "test_{}"
+        table_name = alternator.consts.TABLE_NAME
+        items_size = 10
+
+        if not self.alternator.is_table_exists(node=node, table_name=table_name):
```

Comment:
we should create the table only when `pre_fill=True`, during the upgrade we should never create the table.